nupic/engine/__init__.py

```diff
@@ -729,6 +729,13 @@ def inspect(self):
     from nupic.analysis import inspect    
     inspect(self)
 
+  @staticmethod
+  def registerRegionPackage(path):
+    """
+    @doc:place_holder(Network.registerRegionPackage)
```

Comment:
Would be good to have a nice doc string here since users will actually be calling this function